src/Utils/Logger.ts

```diff
@@ -20,11 +20,21 @@ export class Logger {
   outputChannel: vscode.OutputChannel;
   firstFocus: boolean;
 
-  constructor() {
+  private static logger: Logger|null = null;
+
+  private constructor() {
     this.outputChannel = vscode.window.createOutputChannel('ONE-VSCode');
     this.firstFocus = true;
   }
 
+  public static getInstance(): Logger {
```

Comment:
Note for the other reviewers

https://github.com/Samsung/ONE-vscode/issues/662#issuecomment-1128638648
> (@hyunsik-yoon)  When we call new util.Logger(), new output channel is created. So we should consider, e.g., singleton for Logger.